### PR TITLE
Enable mcg based interop support for native compilation.

### PR DESCRIPTION
**Pull request by tijoytom**
Currently only enabled for windows since mcg is not yet
coreclr ready.Usage:'dotnet compile-native --enableinterop".
Mcg is assumed to be in current directory , this will change
once we package mcg and it's dependencies as nuget packages.

### CODE REVIEW

**Comment by tijoytom** [comment]
Does anyone know why it it keeps on showing "there are conflicts that must be resolved". There can't be any since i just created a new fork and did a pull before applying my changes. Also looking at git log i do not see any commit which is conflicting.

**Comment by brthor** [comment]
@tijoytom I recommend doing a rebase on master. There are likely some merge conflicts

**Comment by tijoytom** [comment]
Thanks , rebasing didn't  help , i had to pull from upstream and manual merge solution file.

**Comment by brthor** [comment]
I typically do this:
git fetch upstream
git rebase -i upstream/master

~* Fix Merge Conflicts *~

git rebase --continue

**Comment by piotrpMSFT** [comment]
@dotnet-bot test this please.

**Comment by tijoytom** [comment]
Guess this looks good? CentOS Debug says 'No test results '. Let me know if i can merge this?

**Comment by piotrpMSFT** [comment]
@tijoytom this needs to be PRd into rel/1.0.0. Anything merged into master is on the train for the next release but won't make 1.0.0.

**Comment by tijoytom** [comment]
@piotrpMSFT , I think we don't probably need it for 1.0.0 , it's half baked now and definitely wont work on Linux or OSX.

File: packaging/debian/dotnet-debian_config.json [hunk 1]
```diff
@@ -44,6 +44,7 @@
         "bin/dotnet-compile-csc" : "usr/bin/dotnet-compile-csc",
         "bin/dotnet-compile-fsc" : "usr/bin/dotnet-compile-fsc",
         "bin/dotnet-compile-native" : "/usr/bin/dotnet-compile-native",
+        "bin/dotnet-mcg": "usr/bin/dotnet-mcg",
```
Comment (brthor) [inline packaging/debian/dotnet-debian_config.json]: This also needs to be added to 

packaging/debian/dotnet-nightly-debian_config.json

**Pull request merged**